process/smartContract/process.go

```diff
@@ -441,14 +445,20 @@ func (sc *scProcessor) DeploySmartContract(
 		return process.ErrWrongTransaction
 	}
 
+	txHash, err := core.CalculateHash(sc.marshalizer, sc.hasher, tx)
```

Comment:
Optional - why not using `computeTransactionHash()` already defined?

Reply:
deleted that function